tools/rdm/TestDefinitions.py

```diff
@@ -2959,9 +2959,10 @@ def OldValue(self):
 
   def VerifyResult(self, response, fields):
     if response.command_class == PidStore.RDM_SET:
-      self.SetProperty('set_device_hours_supported',
-                       response.WasAcked())
-
+      set_unsupported = (
```

Comment:
Would this not be clearer:
set_supported = response.WasAcked() or (response.nack_reason != RDMNack.NR_UNSUPPORTED_COMMAND_CLASS)

Then you can do:
self.SetProperty('set_device_hours_supported', set_supported)